meetings/2022-12-22.md

```diff
@@ -53,6 +57,51 @@ URL: https://github.com/solid/notifications/pull/135
 
 >ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.
 
+* RG: We didn't specify how does the request and response work when you want to modify the subscription
+* SC: Not in this PR, we still didn't reach consensus with this feature. We might need an issue for it.
+* RG: I think we have an issue for it.
+* eP: is there something problematic in 135 if we don't include updating now?
+* RG: No but it will have to do with the data model (eventually). I don't see a problem as another thing but it should be part of the next release.
```

Comment:
```suggestion
* RG: No, but it will have to deal with the data model (eventually). I don't see a problem as another thing, but it should be part of the next release.
```